AppController/djinn.rb

```diff
@@ -5029,8 +5029,7 @@ def is_port_already_in_use(port_to_check)
   # Returns:
   def is_port_assigned(port_to_check)
     @pending_appservers.each { |appserver, _|
-      host, port = appserver.split(":")
-      next if @my_private_ip != host
```

Comment:
Each appengine controller only keeps track of their own pending instances. The keys in `@pending_appservers` are "project_id:port".